politeiawww/api/cms/v1/v1.go

```diff
@@ -391,3 +397,15 @@ type PayInvoices struct{}
 
 // PayInvoicesReply will be empty if no errors have occurred.
 type PayInvoicesReply struct{}
+
+// PaymentInformation contains information for each invoice's payout.
```

Comment:
```suggestion
// PaymentInformation contains information for each invoice's payout. A payout
// might be a single transaction or it might include multiple transactions.
```

This is something that I was confused about until I read the code, so it might be helpful to state it explicitly.  I assumed a payout would be a single transaction at first.